addon/-private/validators.js

```diff
@@ -64,8 +64,15 @@ export function createShapeValidator(validators) {
       return error;
     }
 
+    // Ensure the value is non-null so that we won't accidentally apply get to a null value
+    if (value === null) {
+      return [`Expected a non-null object but received null`, context];
```

Comment:
Slight tweak to make the error message a bit more consistent and clear.

```suggestion
return [`Expected value to be a non-null object but received null`, context];
```

Reply:
Fixed